pkg/apis/componentconfig/edgecore/v1alpha1/validation/validation.go

```diff
@@ -63,8 +64,7 @@ func ValidateModuleEdged(e v1alpha1.Edged) field.ErrorList {
 	}
 	allErrs := field.ErrorList{}
 	if e.NodeIP == "" {
-		allErrs = append(allErrs, field.Invalid(field.NewPath("NodeIp"), e.NodeIP,
-			"Need sed NodeIP"))
+		klog.Warningf("NodeIP is empty , use default ip which can connect to cloud.")
```

Comment:
Thanks for the PR :) In fact, if users haven't specifed the NodeIP, edgecore will set it to local ip by default, so do we need to care it?

Reply:
yes，edgecore will set it to local ip by default. 
so I removed the error "Need sed NodeIP", and fix the cloudstream to let  it get the default local correctly

Reply:
Sorry for the delay, sounds good, would you like squash the commits?

Reply:
> Sorry for the delay, sounds good, would you like squash the commits?

done. 
 (thanks for the review and sorry for delay)